index.html

```diff
@@ -454,6 +391,66 @@ <h4 id="git-linux">Linux</h4>
   </div>
 </div> <!-- End of 'Git' section. -->
 
+<div id="editor"> <!-- Start of 'editor' section. -->
+  <h3>Text Editor</h3>
+
+  <p>
+    When you're writing code, it's nice to have a text editor that is
+    optimized for writing code, with features like automatic
+    color-coding of key words.  The default text editor on Mac OS X and
+    Linux is usually set to Vim, which is not famous for being
+    intuitive.  if you accidentally find yourself stuck in it, try
+    typing the escape key, followed by ':q!' (colon, lower-case 'q',
+    exclamation mark), then hitting Return to return to the shell.
+  </p>
+
+  <div class="row">
+    <div class="col-md-4">
+      <h4 id="editor-windows">Windows</h4>
+      <p>
+	nano is a basic editor and the default that instructors use in the workshop.
+        To install it, 
+        download the <a href="{{site.swc_installer}}">Software Carpentry Windows installer</a>
```

Comment:
The source index.html goes back and forth between tab and space indents.  I don't care which we use (although I'd rather we were consistent), but we should at least be consistent within a single paragraph ;).  Here you use a tab for the first line and spaces for the rest (the original used tabs for all the paragraph lines here, but spaces for the tag lines :p).